superset/views/core.py

```diff
@@ -836,6 +836,16 @@ def shortner(self):
 class Superset(BaseSupersetView):
     """The base views for Superset!"""
 
+    @expose('/rest_actions', methods=['POST'])
+    def restActions(self):
```

Comment:
we follow `_` separated function names, please modify as `rest_actions`to make it consistent.

Reply:
Also, shouldn't be the name as `execute_rest_action`?

Reply:
done